packages/databases-collections/src/index.ts

```diff
@@ -57,6 +61,19 @@ export const DropNamespacePlugin = registerHadronPlugin(
   }
 );
 
+export const RenameCollectionPlugin = registerHadronPlugin(
+  {
+    name: 'RenameCollectionPlugin',
+    component: MappedRenameCollectionModal,
+    activate: activateRenameCollectionPlugin,
+  },
+  {
+    dataService:
+      dataServiceLocator as typeof dataServiceLocator<'renameCollection'>,
+    toastService: toastProviderLocator,
```

Comment:
As I mentioned in another comment, let's not do that, services are only needed for platform specific stuff, you can import and call `openToast` directly, it's a method that works in every environment